tests/test_multipart_installation.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from fastapi import FastAPI, Form
+
+
+def test_incorrect_multipart_installed(monkeypatch):
```

Comment:
Not too sure if this is the best way to detect it. This test will fail if the user doesn't have any kind of `multipart` installed. Maybe we skip it if we can't import `multipart`? thoughts? 

Alternatively, we add our own mock `multipart` package and change the sys path so that it gets imported when the code is run?

Reply:
Also, I'd like to maybe add more specific assertions for the error message possibly as they both raise `RuntimeError`